src/process/ProcessManagerImpl.cpp

```diff
@@ -126,27 +126,45 @@ class ProcessExitEvent::Impl
 size_t
 ProcessManagerImpl::getNumRunningProcesses()
 {
-    return gNumProcessesActive;
+    return gNumProcessesActive -
```

Comment:
Now that the meaning of `gNumProcessesActive` has changed (i.e. it includes processes that are "being killed"), I wanted to double check if its usage in `ProcessManagerImpl::maybeRunPendingProcesses` is right. `gNumProcessesActive` is used to throttle pending processes:
https://github.com/stellar/stellar-core/blob/f3e45d1c3f1f5ea7be221e784d7ab4f292ac2dd0/src/process/ProcessManagerImpl.cpp#L842
I think it would be safer to use `getNumRunningProcesses` in that function?

Reply:
Fixed.